File: Account.py
class A:
    def __init__(self, a, b):
        self.a = a
        self.b = b

    def sum(self):
        return self.a + self.b


class B(A):
    def __init__(self, a, b, c):
        # call the parent class constructor
        super().__init__(a, b)
        self.c = c

    # method overriding
    def sum(self):
        return self.a + self.b + self.c


obj = B(1, 2, 3)

print(obj.a, obj.b, obj.c)
print(obj.sum())

File: Account.js
class A {
    constructor(a, b) {
        this.a = a;
        this.b = b;
    }

    sum() {
        return this.a + this.b;
    }
}

class B extends A {
    constructor(a, b, c) {
        // calling the parent class constructor
        super(a, b);
        this.c = c;
    }

    sum() {
        // calling the parent class method
        return super.sum() + this.c;
    }
}

const obj = new B(1, 2, 3);

console.log(obj.sum()); // 3